### PR TITLE
feat: release pss-baseline-v2022 202301.0

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

File: bundles/pss-baseline-v2022/README.md [hunk 1]
```diff
@@ -26,31 +26,26 @@ Controls](https://kubernetes.io/docs/concepts/security/pod-security-standards/#b
...
-Details: https://kpt.dev/reference/cli/pkg/get/
+### (Optional) Preview the policy constraints with kubectl:
+```shell
+kubectl kustomize https://github.com/GoogleCloudPlatform/acm-policy-controller-library.git/bundles/pss-baseline-v2022
+```
 
-### View package content
-`kpt pkg tree pss-baseline-v2022`
-Details: https://kpt.dev/reference/cli/pkg/tree/
+### Apply the policy constraints with kubectl:
```
Comment (g-awmalik) [inline bundles/pss-baseline-v2022/README.md:36]: Are there instructions to set enforcement action to `deny`? I'm not sure if I see that.

File: bundles/pss-baseline-v2022/sysctls.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  labels:
+    policycontroller.gke.io/bundleName: pss-baseline-v2022
+  annotations:
+    policycontroller.gke.io/constraintData: |
+      "{
+        bundleName: 'pss-baseline-v2022',
+        bundleDisplayName: 'Pod Security Standards Baseline v2022',
+        bundleLink: 'https://github.com/GoogleCloudPlatform/acm-policy-controller-library/tree/master/bundles/pss-baseline-v2022',
+        bundleVersion: '202212.0',
+        bundleDescription: 'Use the Pod Security Standards Baseline v2022 policy bundle with Anthos Policy Controller to to achieve many of the protections of the Kubernetes Pod Security Standards Baseline policy.',
```
Comment (g-awmalik) [inline bundles/pss-baseline-v2022/sysctls.yaml:27]: repetitive "to"

**Review by g-awmalik — APPROVED** [review]

File: bundles/pss-baseline-v2022/README.md (hunk 1 above)
Comment (apeabody) [reply bundles/pss-baseline-v2022/README.md:36]: Yeah, they can be found on the more information link: https://cloud.google.com/anthos-config-management/docs/how-to/using-pss-baseline#enforce_pod_security_standards_baseline_policies

File: bundles/pss-baseline-v2022/sysctls.yaml (hunk 2 above)
Comment (apeabody) [reply bundles/pss-baseline-v2022/sysctls.yaml]: Great catch!

**Pull request merged**